app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -130,26 +135,45 @@ class ConversationViewModel @Inject constructor(
     fun sendAttachmentMessage(attachmentBundle: AttachmentBundle?) {
         viewModelScope.launch {
             attachmentBundle?.let {
-                appLogger.d("> Attachment for conversationId: $conversationId has size: ${attachmentBundle.rawContent.size}")
                 conversationId.run {
```

Comment:
I know you didn't do this, but what is the point of this `conversationId.run` ? I mean: if we remove it, the code will have a 1-less tab, and replacing `this` by `conversationId` in 2 places is not such a big problem IMO